src/lib.rs

```diff
@@ -256,6 +259,28 @@ where
   _p: PhantomData<(C1, C2)>,
 }
 
+/// Final randomized fold of `RecursiveSNARK`
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(bound = "")]
+pub struct RandomLayerRecursive<E1, E2>
```

Comment:
Nit: should we call this `RecursiveSNARKRandomized`?

Reply:
right now, this is _only_ the random information. The CompressedSNARK prover needs access to the original RecursiveSNARK _and_ this random layer. I could make a larger `RecursiveSNARKRandomized` structure that includes both maybe? (Does that make sense?)